### DISCUSSION TITLE
Web-xiangqiJS | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#329] [Announcements]
# Web-xiangqiJS | Zi-Zi's Journey

弘扬中华优秀传统文化！

https://gz-metal-cell.github.io/2024/04/15/Web-xiangqiJS/

**Comment by promefire** [comment]
😲太强了Metal哥😲

**Reply by GZ-Metal-Cell** [reply]
😭导师再不给我审完稿我🐟都要摸烂了😭

**Reply by promefire** [reply]
😭向Metal哥学习😭写完论文就能放心摸鱼了😭

**Reply by GZ-Metal-Cell** [reply]
😭伟哥那才叫方心摸鱼😭我这叫摆烂😭
